alternate/Tenkuu Danzai Skelter Heaven (alternate).ass

```diff
@@ -26,213 +28,220 @@ Style: OPTL,Monotype Corsiva,22,&H00F8F091,&H000000FF,&H00000000,&H7F000000,-1,0
+Dialogue: 0,0:12:29.19,0:12:30.40,Default,,0,0,0,,I...
+Dialogue: 0,0:12:31.10,0:12:34.83,Default,,0,0,0,,I want to know my life.
+Dialogue: 0,0:12:35.30,0:12:39.07,Default,,0,0,0,,The company is responsible for your bodies.
+Dialogue: 0,0:12:39.07,0:12:40.54,Default,,0,0,0,,There's nothing to worry about.
+Dialogue: 0,0:12:41.87,0:12:45.48,Default,,0,0,0,,Just, there remains to be further research to be done on your lifespan.
+Dialogue: 0,0:12:46.78,0:12:50.52,Default,TLC,0,0,0,,Since you're the first generation, you are the research data.{Is that some sort of term there? 第一セライ or something?}{no idae}
+Dialogue: 0,0:12:51.52,0:12:53.25,Default,,0,0,0,,The pointless talking is finished.
+Dialogue: 0,0:12:53.25,0:12:55.45,Default,,0,0,0,,It's okay for you to follow orders.
+Dialogue: 0,0:12:55.45,0:12:56.89,Default,,0,0,0,,Good?
+Dialogue: 0,0:12:56.89,0:12:58.66,Default,,0,0,0,,I understand.
+Dialogue: 0,0:12:59.76,0:13:01.06,Default,,0,0,0,,That's good.
+Dialogue: 0,0:13:01.06,0:13:03.43,Default,,0,0,0,,Taichou's order is absolute.
+Dialogue: 0,0:13:03.43,0:13:06.43,Default,,0,0,0,,You really are always honest, huh?
+Dialogue: 0,0:13:07.03,0:13:07.70,Default,,0,0,0,,Eh?
+Dialogue: 0,0:13:19.41,0:13:21.75,Default,,0,0,0,,Rin, get a hold of yourself!
+Dialogue: 0,0:13:25.42,0:13:27.12,Default,,0,0,0,,I'm okay, I'm okay.
+Dialogue: 0,0:13:27.12,0:13:28.12,Default,,0,0,0,,I can still go!
+Dialogue: 0,0:13:28.45,0:13:30.49,Default,,0,0,0,,Rin... no.
+Dialogue: 0,0:13:30.49,0:13:31.76,Default,,0,0,0,,All units, retreat.
+Dialogue: 0,0:13:33.12,0:13:35.23,Default,TLC,0,0,0,,We will finish it off.
+Dialogue: 0,0:13:35.23,0:13:38.80,Default,,0,0,0,,When its weakness is known, we'll attack this guy and destroy it!
+Dialogue: 0,0:13:38.80,0:13:40.73,Default,,0,0,0,,Funagai! Are you crazy?
+Dialogue: 0,0:13:42.23,0:13:43.90,Default,,0,0,0,,This was just work.
+Dialogue: 0,0:13:45.67,0:13:48.71,Default,,0,0,0,,Thinking that, I somehow did it.
+Dialogue: 0,0:13:49.47,0:13:50.28,Default,,0,0,0,,But...
+Dialogue: 0,0:13:51.54,0:13:52.68,Default,,0,0,0,,They...
+Dialogue: 0,0:13:53.31,0:13:54.41,Default,,0,0,0,,were laughing.
+Dialogue: 0,0:13:56.11,0:13:58.38,Default,,0,0,0,,I am... I am...
+Dialogue: 0,0:14:08.29,0:14:09.26,Default,TM,0,0,0,,Ah!
+Dialogue: 0,0:14:13.47,0:14:15.93,Default,,0,0,0,,What are you doing, Rin?
+Dialogue: 0,0:14:15.93,0:14:20.31,Default,,0,0,0,,I didn't follow your orders because I was made like that!
+Dialogue: 0,0:14:20.81,0:14:23.94,Default,,0,0,0,,It's because I loved you!
+Dialogue: 0,0:14:26.85,0:14:27.88,Default,,0,0,0,,That's it.
+Dialogue: 0,0:14:29.01,0:14:30.95,Default,,0,0,0,,I'm going on ahead!
+Dialogue: 0,0:14:36.39,0:14:37.26,Default,,0,0,0,,Don't go!
+Dialogue: 0,0:14:37.89,0:14:39.79,Default,,0,0,0,,Stop!
+Dialogue: 0,0:14:57.98,0:14:59.18,Default,,0,0,0,,I got in!
+Dialogue: 0,0:15:01.18,0:15:03.15,Default,,0,0,0,,Rin-san, I'm going!
```

Comment:
I find it hilarious that the other subs misheard this as "nii-san". Don't change it, though, because then it would seem like we're copying them.